### DISCUSSION TITLE
Friendly Fire

### DISCUSSION THREAD

**Discussion by thevibingcat** [redeclipse/base#1356] [General]
What will happen if you kill too many teammates? if the developers don't want teamkilling to happen why don't they just remove friendly fire altogether? friendly fire can be a serious problem sometimes. if you place a laser mine against a wall, 90% of the time a teammate (who is a bot) will walk into it. this also messes up my new joke strategy of kamikaze (run up to an enemy with a grenade out, and press K), which in some cases I will end up exploding teammates instead of my target. in my opinion, friendly fire is an annoying feature.

**Comment by Calinou** [comment]
In Cube 2 engine games, friendly fire is enabled by default because it helps balance weapons with area-of-effect damage (flamer, grenade, rocket). It also encourages you to think more before you shoot :slightly_smiling_face: 

There's an argument for disabling friendly fire for mines, but I still think it should be limited in scope somehow (for instance, only bots would be immune to it). I think players are already immune to friendly fire from bots.

**Reply by thevibingcat** [reply]
I think large sources of damage (e.g., rocket launcher, plasma inductor [mouse2], flamethrower, etc.) should have friendly fire, whereas other sources (rifle, shotgun, etc.) should not.
The only time I do get use out of friendly fire is when I kill another teammate (bot btw) for a rocket launcher (bots use large damage sources poorly sometimes).
